src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Export.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    public enum ConvertEnum
+    {
+        Default,
+        ToLower,
+        ToUpper,
+        Numeric
+    }
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Property | AttributeTargets.Method | AttributeTargets.Field,
+            AllowMultiple = true, Inherited = false)]
+    public class ExportAttribute : Attribute
```

Comment:
The name is odd if we are using it for enums only

Reply:
Ok will change it to EnumExport.

Reply:
Addressed in recent commits